.github/workflows/verify.yml

```diff
@@ -74,6 +77,12 @@ jobs:
           ruby-version: ${{ matrix.ruby }}
           bundler-cache: true
 
+      - name: Update Rails version
+        run: |
+          ruby -pi.bak -e "gsub(/gem 'rails',\s*['\"].+['\"]?/, \"gem 'rails', '${{ matrix.rails }}'\")" Gemfile
```

Comment:
This wouldn't work for the other repos that just use `gem 'rails'`, or `gem "rails"` - but I guess you can come to that bridge when you come to it for the other repos :+1: